src/doc/unstable-book/src/language-features/lang-items.md

```diff
@@ -278,6 +276,7 @@ the source code.
   - `generator`: `libcore/ops/generator.rs`
 - Other
   - `coerce_unsized`: `libcore/ops/unsize.rs`
+  - `unsafe_cell`: `libcore/cell.rs`
```

Comment:
I think we should move this to the bottom or to the first item in `Other` since it is between two `ops` items.

Reply:
I created a new category and moved `UnsafeCell` to it, together with `ManuallyDrop`.